### PR TITLE
update french.txt

### PR DESCRIPTION
**Pull request by TheNoobestNoob**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by TheNoobestNoob** [comment]
it's quite hard to translate the next ones into french, so i prefer not to touch them

**Review by Chairn — CHANGES_REQUESTED** [review]

**Comment by def-** [comment]
No worries, don't have to translate all, any improvement helps :)

File: data/languages/french.txt [hunk 1]
```diff
@@ -1572,37 +1573,37 @@ Loading race demo files
 == Chargement des fichiers de demo de course
 
 Play the current demo
-== 
+== Lecture de la démo en cours
```
[the remark this answers is not in the document]
Comment (TheNoobestNoob) [reply data/languages/french.txt:1576]: thanks, how can i "edit" it? xd
do I have to do another commit or smth?

[the remark this answers is not in the document]
Comment (Chairn) [reply data/languages/french.txt:1576]: Did you edit directly on github website?

[the remark this answers is not in the document]
Comment (def-) [reply data/languages/french.txt:1576]: Yes, add another commit, you can inline edit the change directly in the editor here on github.com.

[the remark this answers is not in the document]
Comment (def-) [reply data/languages/french.txt:1576]: If you want to be fancy and use command line, then you can make your change and add it with `git commit -a --amend` and then `git push -f`.

**Review by Chairn — APPROVED** [review]

File: data/languages/french.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheNoobestNoob) [reply data/languages/french.txt]: alright, should be good now (I used github's editor lol).

**Pull request merged**